README.md

```diff
@@ -52,44 +62,85 @@ After you have cloned the repository:
 
 Now you can visit [`localhost:4000`](http://localhost:4000) from your browser.
 
-### Configure external API access keys
+#### Configuration
+
+The app’s configuration are following the [Twelve Factor](https://12factor.net/config)
+methodology, which means they are stored in the `environment`. You may use `source`,
+[`direnv`](https://direnv.net), [`nv`](https://github.com/jcouture/nv) or a custom script of your own.
```

Comment:
Since we deploy on Heroku and it's a deployment platform familiar to most developers new and experienced, let's embrace their ecosystem and make the [Heroku CLI ](https://devcenter.heroku.com/articles/heroku-cli) a prerequisite. So the recommended way for starting up the app with a configuration file would be with `heroku local`. [This would require adding a Procfile](https://devcenter.heroku.com/articles/heroku-local).